k8s/airflow/stage-airflow-app.yaml

```diff
@@ -128,9 +128,25 @@ metadata:
  name: airflow-svc
  namespace: pipeline
 spec:
+ type: NodePort
  ports:
   - port: 8080
     protocol: TCP
     targetPort: 8080
+    nodePort: 30036
```

Comment:
Is it possible to change this to 31036 for staging in accordance to [this doc](https://docs.google.com/document/d/1r0wJLxr2jrtjk8oqnv-N5DBNcORYENeWQntDIjwhVLU/edit#)